src/Crud/PagesRequestFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace Biig\Melodiia\Crud;
+
+use Symfony\Component\HttpFoundation\Request;
+
+class PagesRequestFactory
```

Comment:
This class is definitely a goog idea, but using static method without interface, you enforce its usage and make it useless.
Please make it service with an interface so users can define another one.

Reply:
Done

need to add user service auto-registration if the interface is implemented

Reply:
I don't get your point. I don't think this is required. If needed the user just override the service.